packages/subiquity_client/lib/src/server.dart

```diff
@@ -11,6 +11,25 @@ enum ServerMode { LIVE, DRY_RUN }
 /// @internal
 final log = Logger('subiquity_server');
 
+/// Vocabulary type that combines an [InternetAddress] and a [port] number
+class Endpoint {
```

Comment:
could we have the `Endpoint` class in a separate `endpoint.dart` file so the client and server don't need to import each other?

Reply:
Silly me not noticing the server was imported into the client automatically. Sorry about that. Fixed now.